src/pudl/etl/static_assets.py

```diff
@@ -86,18 +82,14 @@ def static_ferc1_tables():
     """Compile static tables for FERC1 for foriegn key constaints.
 
     This function grabs static encoded tables via :func:`_read_static_encoding_tables`
-    as well as two static tables that are non-encoded tables (``ferc_accounts`` and
-    ``ferc_depreciation_lines``).
+    as well as two static tables that are non-encoded tables (``ferc_accounts``).
     """
     static_table_dict = _read_static_encoding_tables("static_ferc1")
     static_table_dict.update(
         {
             "ferc_accounts": FERC_ACCOUNTS[
                 ["ferc_account_id", "ferc_account_description"]
             ],
-            "ferc_depreciation_lines": FERC_DEPRECIATION_LINES[
```

Comment:
Should we keep the `FERC_DEPRECIATION_LINES` around?

Reply:
I think this static table has been replaced by an XBRL table. See [this commit](https://github.com/catalyst-cooperative/pudl/commit/e7814aec8af9469bc168f0b6fb735c889ae9559f).